cssom-1/Overview.bs

```diff
@@ -1541,16 +1544,12 @@ must be run:
      </ol>
    </ol>
  </ol>
-</div>
 
-A style sheet referenced by an <a>xml-stylesheet processing instruction</a> using the rules in this section, in the context of
-the {{Document}} of an <a>XML parser</a> is said to be
-<a>a style sheet that is blocking scripts</a> if the <code>ProcessingInstruction</code>
-<a>node</a> was created by that {{Document}}'s parser, and the style sheet was
-enabled when the node was created by the parser, the last time the
-<a>event loop</a> reached step 1, the node was in that Document,
-and the user agent hasn't given up on loading that particular style sheet
-yet. A user agent may give up on such a style sheet at any time.
+If <var>node</var> <a>contributes a script-blocking style sheet</a>, then user agents must <a
+for=list>append</a> <var>node</var> to its <a>node document</a>'s <a for=Document>script-blocking
+style sheet set</a>.
+
+</div>
```

Comment:
Wanted to keep this text inside the algorithm just so the variable highlighting works, but more comfortable following the structure of this spec _before_ this PR, where this text is outside the actual numbered steps.